frontend/layouts/default/Newsletter.tsx

```diff
@@ -1,86 +1,92 @@
 import {
    Box,
-   BoxProps,
    Button,
    FormControl,
-   FormControlProps,
-   FormErrorMessage,
-   FormLabel,
-   FormLabelProps,
-   forwardRef,
    HStack,
    Input,
-   InputProps,
    Stack,
-   StackProps,
    Text,
-   TextProps,
+   Flex,
+   Link,
 } from '@chakra-ui/react';
 import { faCircleCheck } from '@fortawesome/pro-solid-svg-icons';
+import { faEye } from '@fortawesome/pro-solid-svg-icons/faEye';
```

Comment:
Hmm.. is the large JS increase because of these two different import styles? Hard to imagine... but it's an easy experiment to combine them and see (perhaps going with whatever is more common in this repo)

Reply:
I combined these in [0cc257f](https://github.com/iFixit/react-commerce/pull/1342/commits/0cc257fd80d9c7742356d33f63dffa7d2178cff7), and the bundle stats report increased from `+11 B` to `+307 B` on the global bundle size, so it seems like importing these separately is better, even though we have combined imports everywhere else in this repo. 

Correct me if I'm wrong, but isn't `+11 B` (bytes) extremely tiny in comparison to the whole build? Where are you getting the +10% increase from?

Reply:
Yes, +300b is tiny, however, right below that in the same comment we have stats for each page:
![image](https://user-images.githubusercontent.com/26855/219438917-d5e06756-0c0b-4a7b-81f8-c7bfbaa4754f.png)

Reply:
I ran the webpack analysis on my branch and main and they ended up being very similar:
**On my branch:**
<img width="1506" alt="Screen Shot 2023-02-17 at 10 59 15 AM" src="https://user-images.githubusercontent.com/43479402/219763045-dbb1c5f7-4f1e-4d11-a0f4-d32dffb39a03.png">

**On Main:**
<img width="1506" alt="Screen Shot 2023-02-17 at 10 56 39 AM" src="https://user-images.githubusercontent.com/43479402/219763207-99555877-33b2-4faa-844a-7dd4f78c544a.png">


The only difference I can really see is the increase in sdk.ts in 647.js. That seems like something that could touch all of the above pages.

Is there a way to re-run the bundle analysis from CI? It didn't seem to update after [d09523d](https://github.com/iFixit/react-commerce/pull/1342/commits/d09523df726209f846f29f9d7f814836cccdbe76).

Reply:
> Is there a way to re-run the bundle analysis from CI? It didn't seem to update after

Looks like it updated now:
![image](https://user-images.githubusercontent.com/26855/219780195-f4417b0f-2ddd-40b6-9bc1-f9babdac466e.png)

It looks like your analysis is comparing the *server bundles* and not the client bundles (I think).

Reply:
@aburke07 I reran the Next.js Bundle Analysis. Here's where I found the job to rerun it on this branch: https://github.com/iFixit/react-commerce/actions/workflows/nextjs_bundle_analysis.yml

Reply:
Hmm...  I wonder if it has to do with there being conflicts on this branch.